### ISSUE TITLE
Review/feedback to consider

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/lake-temp-lstm-static-data-release#28]
An issue to collect reviewer and collaborator feedback.

**Comment by hcorson-dosch-usgs** [comment]
You have a nice snippet in there explaining the depth discretization for the EALSTM modeling:
> Lakes were discretized into a series of depth intervals. The top 10 meters of the lake were discretized into 0.5 m intervals. From 10 to 20 m, 1 m intervals were used. From 20 to 40 m, 2 m intervals were used. From 40 to 80 m, 5 m intervals were used. 

I wonder if we should explicitly state somewhere that these same depth intervals were used for the exported temperature predictions, including those generated by GLM?

If that was added to the process description, it could also be referenced in the attribute description for `depth` for the prediction netCDFs

**Comment by hcorson-dosch-usgs** [comment]
Also I was wondering if we should note somewhere the exact version of GLM used - `GLM version 3.2.0a8` - maybe in the modeling environment section?

**Comment by padilla410** [comment]
100% agree with Hayley on adding the GLM model version

**Comment by AndyMcAliley** [comment]
In the SB item, under "Related External Resources" there are two lines, both of which show the same link to a DOI. I'm not sure why there are two lines there. Also, the description field of "Type: webLink" says "a single comma-delimited file and a single zipped shapefile." What does that description refer to?

**Comment by lindsayplatt** [comment]
OK, so I found where I can edit the description (and I will do that along with @hcorson-dosch-usgs's two suggestions above very soon) but I think the FGDC metadata in the `meddle` package will automatically create these two external resources. I am not sure whether they are necessary but if you CTRL+F [here in the template](https://github.com/USGS-R/meddle/blob/main/inst/extdata/FGDC_template.mustache) for `{{doi}}` there are two places that appear.

**Comment by lindsayplatt** [comment]
Adding to my previous statement, those fields appear in both XML files for the MO reservoir release and the lake surface temp release, but are not visible in the item. My guess is that they are there as more of an internal view and won't be visible publicly. I will still update the file format description, but I don't think we need to worry about it otherwise.

**Comment by lindsayplatt** [comment]
FYI, I have a PR coming soon but the slow downs noted in the Tallgrass channel on Teams today have been a big blocker. I can't finish the SB upload through the pipeline or even push my changes to GitHub without a timeout. As such, I have manually uploaded my updated XML file with the changes noted above so that I can lock the SB item for review. I'll try push again later tonight but may be submitting my PR on Thursday and hope that things have been resolved by then.
